extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -247,7 +249,7 @@ class AmpVideoIframe extends AMP.BaseElement {
         this.postIntersection_(messageId);
         return;
       }
-      dev().assert(false, `Unknown method '${methodReceived}`);
```

Comment:
This message looks less useful without also logging the `methodReceived` variable. Was this to stop the "logging non-string" warning?

Reply:
Yup. I realized that `assert` takes `var_args` so the string could still be interpolated. Changing.